articles/tokens/access-token.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Access Token
+description: This page explains an overview about Auth0 access tokens.
+---
+
+# Access Token
+
+## Overview
+
+The Access Token, commonly referred to as `access_token` in code samples, is a credential that can be used by a client to access an API. The `access_token` should be used as a `Bearer` credential and transmitted in an HTTP `Authorization` header to the API. Auth0 uses access tokens to protect access to the Auth0 Management API.
+
+By default, Auth0 generates access tokens in JSON Web Token (JWT) format, an industry standard. JWTs contain three parts: a header, a set of claims, and a signature:
+- The header contains metadata about the type of token and the cryptographic algorithms used to secure its contents.
+- The set of claims contains verifiable security statements such as the identity of the user and the permissions they are allowed.
+- The signature is used to validate that the token is trustworthy and has not been tampered with.
+
+## How to get an access token
+
+Access tokens are issued via [Auth0's OAuth 2.0 endpoints](/api/authentication): `/authorize` and `/oauth/token`. You can use any OAuth 2.0-compatible library to obtain access tokens. If you do not already have a preferred OAuth 2.0 library, Auth0 provides libraries for many language and frameworks that work seamlessly with our endpoints.
+
+* Calls to the Lock widget will return an `access_token` as shown in the [Lock documentation](/libraries/lock).
+* [Examples using auth0.js](https://github.com/auth0/auth0.js).
+* Check the [List of tutorials](/tutorials) to see how to make calls to libraries for other languages/SDKs.
+
+## How to use an access token
+
+Access tokens are typically obtained in order to access user-owned resources. For example, a Calendar application on a mobile phone needs access to a Calendar API in the cloud in order to read the user's scheduled events and create new events.
+
+Such access is requested by the application and granted by the user. For example, when using Lock, access to read and write the user's calendar can be requested by calling `show()`:
+
+```js
+lock.show({
+  responseType: 'token',
+  authParams: {
+    scope: 'read write'
```

Comment:
This is missing the `audience` param with the API audience. Without this param, the request will go thru the legacy pipeline.

Maybe we can replace the usage of lock with a call to `/authorize`. This will be better since Lock still has a lot of caveats when it comes to using `audience` (e.g. support for 3rd party hosted lock is not yet completed)